module/LearnZF2AjaxImageGallery/src/LearnZF2AjaxImageGallery/Form/AjaxImageUploadForm.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace LearnZF2AjaxImageGallery\Form;
+
+use Zend\Form\Form;
+use Zend\Form\Element;
+
+class AjaxImageUploadForm extends Form
+{
+    public function __construct()
+    {
+        parent::__construct("content");
+        $this->setAttribute('method', 'post');
+
+        $this->add([
+            'name' => 'imageUpload',
+            'type'  => 'File',
+            'attributes' => [
+                'class' => 'imgupload',
+                'id' => 'imgajax',
+                'multiple' => true,
+            ],
+        ]);
+    }

```

Comment:
as we will use FormElementManager, new getInputFilter() |setInputFilter() methods are not needed in both Model and Form. getInputFilter() is callable by form instance